ClientSide(FrontEnd)/Consumer_Side/src/components/Displayproduct.jsx

```diff
@@ -51,23 +70,23 @@ const Displayproduct = () => {
     <div>
         <div className="color">
         <div className="categories">
-                        <div className="design">
-                            <img src="https://leadwinner.com/shilparamam2/images2/tera/3.jpg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260" alt="" />
+                        <div className="design" onClick={() => categoryFunction(116)}>
+                            <img src="https://i.etsystatic.com/26610308/r/il/723e68/2831141092/il_1588xN.2831141092_dr44.jpg" alt="" />
                         </div>
-                        <div className="flower">
-                            <img src="https://leadwinner.com/shilparamam2/images2/tera/1.jpg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260" alt="" />
+                        <div className="flower" onClick={() => categoryFunction(117)}>
```

Comment:
Please use a single class name and define style, no need to create different class names from each element